examples/ota-provider-app/linux/README.md

```diff
@@ -13,19 +13,19 @@ scripts/examples/gn_build_example.sh examples/ota-provider-app/linux out/debug c
+| -q/--queryImageStatus <updateAvailable \| busy \| updateNotAvailable> | Value for the Status field in the first QueryImageResponse. <br> For all subsequent responses, the value of updateAvailable will be used.                                                                                                                                                                                                                                                                                                                                                                                  |
+| -x/--ignoreQueryImage <num_times_to_ignore>                           | The number of times to ignore the QueryImage Command and not send a response                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| -y/--ignoreApplyUpdate <num_times_to_ignore>                          | The number of times to ignore the ApplyUpdate Request and not send a response                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| -a/--applyUpdateAction <proceed \| awaitNextAction \| discontinue>    | Value for the Action field in the first ApplyUpdateResponse. <br> For all subsequent responses, the value of proceed will be used.                                                                                                                                                                                                                                                                                                                                                                                         |
+| -t/--delayedActionTimeSec <time>                                      | Value in seconds for the DelayedActionTime field in the QueryImageResponse and ApplyUpdateResponse. <br> Value will revert back to 0 seconds on subsequent QueryImage Responses and ApplyUpdate Responses.                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Hmm, actually this is not always true? You don't reset `mDelayedActionTimeSec` until `HandleApplyUpdateRequest` which means the value specified here will keep applying for QueryImageResponse until a path that leads to `HandleApplyUpdateRequest` being called.

For example, if provider was launched with `--queryImageStatus busy --delayedActionTimeSec 300`. Then you'd get:
Requestor->Provider: QueryImage command
Provider->Requestor: QueryImageResponse with Status=Busy, DelayedActionTime=300
Requestor->Provider: QueryImage command
Provider->Requestor: QueryImageResponse with Status=Available, DelayedActionTime=300
Requestor->Provider: ApplyUpdateRequest command
Provider->Requestor: ApplyUpdateResponse with Action=Proceed, DelayedActionTime=300

I don't believe this is the desired behavior? Wonder if we should just have two separate command line args for each respective delayed time?

Reply:
Sure, I'll split this into 2 CLI args then.